### ISSUE TITLE
Issue--How to train this Img2mol model 

### ISSUE THREAD

**Issue by whkaikai** [bayer-science-for-a-better-life/Img2Mol#13]
Hi, OBrink:
When I was testing this model, I met one problem:
![test5](https://user-images.githubusercontent.com/46772304/217140122-9d7cb871-746e-4a8d-a2c9-5e4527ae95c8.png)
![test6](https://user-images.githubusercontent.com/46772304/217140131-16c2e9a4-52aa-4e0a-bc5b-cbe65a409789.png)
I used train model but get wrong predictions.

Do you have any advices to get high accuracy to test these pictures, we want to input these pictures from patents and get right smiles ?
In my personal opinion, it is because there is no such sample when the model is trained, so can I use this Img2mol model that has been trained?

**Comment by fmonta** [comment]
Hi @whkaikai, please have a look @OBrink's detailed answer to your issue in DECIMER. I would simply say that these types of pseudo-molecules are not part of the training set of Img2Mol and could not be expected to be represented by SMILES strings anyhow. I don't think you will have better success with the pre-trained Img2Mol than with DECIMER, but of course, feel free to try.

**Comment by whkaikai** [comment]
Thank you very much, it helps a lot.
